src/cpdetect/compiler_detector/heuristics/pe_heuristics.cpp

```diff
@@ -101,6 +101,17 @@ const std::vector<std::string> msvcRuntimeStrings =
 	toWide(msvcRuntimeString, 4)
 };
 
+const std::vector<PeHeaderStyle> headerStyles = 
```

Comment:
There are many useless trailing whitespaces (see the diff below). Please, remove them ([reasons](https://cs-blog.petrzemek.net/2011-09-09-proc-se-zbavovat-nadbytecnych-bilych-znaku)). I suggest setting your editor to remove them automatically to prevent their appearance in the future.
```diff
diff --git a/src/cpdetect/compiler_detector/compiler_detector.cpp b/src/cpdetect/compiler_detector/compiler_detector.cpp
index c8615660..8fa7da8e 100644
--- a/src/cpdetect/compiler_detector/compiler_detector.cpp
+++ b/src/cpdetect/compiler_detector/compiler_detector.cpp
@@ -532,12 +532,12 @@ ReturnCode CompilerDetector::getAllInformation()
	fileParser.getImageBaseAddress(toolInfo.imageBase);
	toolInfo.entryPointAddress = fileParser.getEpAddress(toolInfo.epAddress);
	toolInfo.entryPointOffset = fileParser.getEpOffset(toolInfo.epOffset);
-
+
	if((toolInfo.overlaySize = fileParser.getOverlaySize()) != 0)
	{
		toolInfo.overlayOffset = fileParser.getDeclaredFileLength();
	}
-
+
	const bool invalidEntryPoint = !toolInfo.entryPointAddress || !toolInfo.entryPointOffset;
	if (!fileParser.getHexEpBytes(toolInfo.epBytes, cpParams.epBytesCount)
			&& !invalidEntryPoint && !fileParser.isInValidState())
diff --git a/src/cpdetect/compiler_detector/heuristics/pe_heuristics.cpp b/src/cpdetect/compiler_detector/heuristics/pe_heuristics.cpp
index d085c184..f1faa966 100644
--- a/src/cpdetect/compiler_detector/heuristics/pe_heuristics.cpp
+++ b/src/cpdetect/compiler_detector/heuristics/pe_heuristics.cpp
@@ -101,7 +101,7 @@ const std::vector<std::string> msvcRuntimeStrings =
	toWide(msvcRuntimeString, 4)
 };

-const std::vector<PeHeaderStyle> headerStyles =
+const std::vector<PeHeaderStyle> headerStyles =
 {
 // {"Unknown",      { 0x0000, 0x0000, 0x0000, 0x0000, 0x0000, 0x0000, 0x0000, 0x0000, 0x0000, 0x0000, 0x0000, 0x0000, 0x0000 }},
	{"Microsoft",    { 0x0090, 0x0003, 0x0000, 0x0004, 0x0000, 0xFFFF, 0x0000, 0x00B8, 0x0000, 0x0000, 0x0000, 0x0040, 0x0000 }},
@@ -611,7 +611,7 @@ void PeHeuristics::getSecuROMHeuristics()
	bool foundSecuROM = false;

	// There must be at least 0x2000 extra bytes beyond the last section,
-   // last data directory, last debug directory and digital signature.
+   // last data directory, last debug directory and digital signature.
	if (peParser.getOverlaySize() >= 0x2000)
	{
		// The entire file must be loaded to memory
@@ -626,7 +626,7 @@ void PeHeuristics::getSecuROMHeuristics()
				foundSecuROM = true;
			if(checkSecuROMSignature(fileData, fileData + loadedLength, loadedLength - (SecuromOffs - 0x0C)))
				foundSecuROM = true;
-
+
			if(foundSecuROM)
			{
				addPacker(DetectionMethod::STRING_SEARCH_H, DetectionStrength::HIGH, "SecuROM");
```